Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php

```diff
@@ -40,8 +40,8 @@ public function fix(\SplFileInfo $file, $content)
 
             // ignore inline docblocks
             if (
+                $prevToken->equalsAny(array(';', '{')) ||
                 ($prevToken->isWhitespace(array('whitespaces' => " \t")) && !$tokens[$index - 2]->isGivenKind(T_OPEN_TAG))
-                || $prevToken->equalsAny(array(';', '{'))
```

Comment:
Will revert then.

PS: In 1.7 Token operates internally on plain string API as I can see.